### PR TITLE
About unarmed parrying..

### PR DESCRIPTION
**Pull request by DeltaFire15**
## About The Pull Request
Yeah this is getting out of hand.
If a lot of people keep making parry armwraps roundstart, and those actually gave them an absurdly good fighting chance in combat, even against antagonists with gear as I have witnessed, it's time to look at those values.
Silicons may be working on a PR to rebalance parrying & combat, but until then, it shouldn't stay in its current state either.

Pugilist and normal unarmed parries now have a cooldown even on success, and stagger slightly when failed. Plus some more changes, as you can see in the files themselves.

## Why It's Good For The Game
Unarmed parries, especially pugilist ones, are WAY too strong right now. This kneecaps them somewhat, hopefully making them not as powerful.

## Changelog
:cl:
balance: Kneecapped pugilist parries somewhat.
balance: Slightly nerfed default unarmed parries.
balance: Slightly nerfed traitor armwrap parries.
fix: Pugilist parries now cannot perfectly defend against projectiles, as they were supposed to.
fix: Some parrying numbers that one would think were in seconds didn't have the SECONDS. I added those.
/:cl:

### CODE REVIEW

**Comment by kiwedespars** [comment]
> If a lot of people keep making parry armwraps roundstart, and those actually gave them an absurdly good fighting chance in combat, even against antagonists with gear as I have witnessed,

it's more of, this is a really cool way to fight people, because taking anything with a weapon down with just your bare hands is always really cool. using actual weapons has been better, and will continue to be better. what antags have been getting beat by armwraps anyways.

**Comment by dzahlus** [comment]
> > If a lot of people keep making parry armwraps roundstart, and those actually gave them an absurdly good fighting chance in combat, even against antagonists with gear as I have witnessed,
> 
> it's more of, this is a really cool way to fight people, because taking anything with a weapon down with just your bare hands is always really cool. using actual weapons has been better, and will continue to be better. what antags have been getting beat by armwraps anyways.

Cultists and Traitors, I've witnessed it multiple times. You can just parry spam and it makes it really really obnoxious to fight people who train with it every round, it is simply way too strong in it's current state and how easy it is to get.

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -220,8 +220,8 @@
 	parry_max_attacks = INFINITY
 	parry_failed_cooldown_duration = 2.25 SECONDS
 	parry_failed_stagger_duration = 2.25 SECONDS
-	parry_cooldown = 0
-	parry_failed_clickcd_duration = 0
+	parry_cooldown = 3 SECONDS
```
Comment (deathride58) [inline code/modules/clothing/gloves/miscellaneous.dm:223]: 3 seconds is a little overkill of a nerf considering this is a traitor item, don't you think?

File: code/modules/mob/living/carbon/human/human_defines.dm [hunk 2]
```diff
@@ -135,16 +136,16 @@
...
 
 	parry_efficiency_considered_successful = 0.01
 	parry_efficiency_to_counterattack = 0.01
 	parry_max_attacks = INFINITY
-	parry_failed_cooldown_duration =  1.5 SECONDS
-	parry_failed_stagger_duration = 0
-	parry_cooldown = 0
-	parry_failed_clickcd_duration = 0.8
+	parry_failed_cooldown_duration =  3 SECONDS
+	parry_failed_stagger_duration = 2 SECONDS
```
Comment (deathride58) [inline code/modules/mob/living/carbon/human/human_defines.dm:146]: These values feel as though they're mixed up with standard unarmed, as above. These values make pugilist parrying fairly unwieldy compared to standard unarmed parrying.

**Comment by silicons** [comment]
im merging this in like 1-2 days @DeltaFire15 i do recommend making sure the values aren't too nerfed for antags as antags and regular parry armwraps use the same data (my fault really) but this is a good patch until next major update is done soonish

**Pull request merged**